cmd/hypper/install.go

```diff
@@ -101,6 +101,7 @@ func addInstallFlags(cmd *cobra.Command, f *pflag.FlagSet, client *action.Instal
 	f.BoolVar(&client.NoSharedDeps, "no-shared-deps", false, "skip installation of shared dependencies")
 	f.Var(enumflag.New(&optionaldepsmode, "option", OptionalDepsModeIds, enumflag.EnumCaseInsensitive),
 		"optional-deps", "install optional shared dependencies [ask|all|none]")
+	f.BoolVar(&client.DryRun, "dry-run", false, "simulate an install")
```

Comment:
Wouldn't it be nicer to have this as a global flag? I think this would also be helpful for removing stuff and updating namespaces.

Reply:
There are some commands (`ls`, `lint`, `shared-dep list`, `repo list`, etc) that may not need it. And since Helm has the flag per command, I went with that too.

Reply:
Ok, reasonable approach.